website/content/docs/agent/telemetry.mdx

```diff
@@ -477,6 +477,7 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.raft.verify_leader`                         | This metric doesn't have a direct correlation to the leader change.  It just counts the number of times an agent checks if it is still the leader or not.  For example, during every consistent read, the check is done.  Depending on the load in the system, this metric count can be high as it is incremented each time a consistent read is completed.                                                                                                                                                                                                                                                                                                                                                                                        | checks / interval                 | Counter |
 | `consul.rpc.accept_conn`                            | Increments when a server accepts an RPC connection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | connections                       | counter |
+| `consul.rpc.rate_limit.exceeded`                    | Increments whenever an RPC is over a configured rate limit. Note: in permissive mode, the RPC will have still been allowed to proceed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | RPCs                              | counter |
```

Comment:
```suggestion
| `consul.rpc.rate_limit.exceeded`                    | Increments whenever an RPC is over a configured rate limit. When Consul is in `permissive` mode, the RPC is allowed to proceed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | RPCs                              | counter |
```
Actually, what is in permissive mode in this context? I assumed that `permissive` is a value that you you can define in some configuration, but I'm not sure. Also, we should just use simple present tense per the style guide.

Reply:
Thanks for fixing my terrible writing! 😆 ❤️  

In this context, the rate limiter is in permissive mode, and there can be multiple limiters involved in a single RPC (e.g. global, ip-based, tenant-based) which are configured independently.